### ISSUE TITLE
Has there been any discussion of an onSecond callback?

### ISSUE THREAD

**Issue by patsplat** [pughpugh/react-countdown-clock#33]
Am investigating how to add sound to the countdown clock.  Has there been any discussion of a callback for 5, 4, 3, 2, 1, etc seconds?

**Comment by macdonst** [comment]
Yeah, I'm also looking to play a sound as the timer counts down. Having a progress event fired would help but having the clock handle playing the sound would probably be best to keep them in sync.

**Comment by patsplat** [comment]
@macdonst your commit would solve the reported issue.  Have you tried PR'ing back to @pughpugh ?
